django/core/management/commands/shell.py

```diff
@@ -139,9 +140,13 @@ def get_auto_imports(self):
         ]
 
         """
+        try:
+            models = apps.get_models()
+        except AppRegistryNotReady:
+            return []
```

Comment:
Should this possibility be noted in the doc string? I think when reading the code it wouldn't matter, but if there's reference documentation generated elsewhere, that may be beneficial. I'm not 100% on what things look like here.

Would it be ideal here if there's a higher verbosity set, that there'd be a log/line telling the user why the imports weren't automatically performed?